app/pages/lab/workflow-viewer/task.cjsx

```diff
@@ -0,0 +1,313 @@
+React = require 'react'
+Resizable = require 'react-component-resizable'
+{TaskName, RequireBox, HelpButton} = require './task-helpers'
+AnswerList = require './answers'
+Sty = require './jsplumb-style'
+
+# keep track of the max z-index
+max_z = 0
+
+# Firefox needs an offset patch for jsPlumb elements (not sure way)
+isFirefox = typeof InstallTrigger != 'undefined'
+
+# timer to detect when resize stops
+timer = null
+
+# Make a task node
+# Takes `plumbId`, `workflow`, `taskKey`, `pos`, and `taskNumber` as a props
+Task = React.createClass
+  displayName: 'Task'
+
+  getInitialState: ->
+    zIndex: @props.taskNumber + 2
+    task: @props.wf[@props.taskKey]
+    endpoints: []
+    zIndex: @props.taskNumber + 2
+    uuid: 0
+    uuids: []
+    me: null
+    width:
+      min: 0
+      max: 0
+      val: 0
+
+  componentDidMount: ->
+    me = React.findDOMNode(@)
+    # make node draggable using jsPlumb
+    drag_options =
+      handle: '.drag-handel'
+      start: @onDrag
+      stop: @onDragStop
+    @props.jp.draggable(@props.plumbId, drag_options)
+    if @state.task.type == 'drawing'
+      ep = @props.jp.addEndpoint(@props.plumbId, Sty.commonT_draw, {uuid: @props.plumbId})
+    else
+      ep = @props.jp.addEndpoint(@props.plumbId, Sty.commonT, {uuid: @props.plumbId})
+    ep.canvas.style['z-index'] = @state.zIndex
+    eps = [ep]
+    # make subtask endpoints
+    if @state.task.subtask
+      ep = @props.jp.addEndpoint(@props.plumbId+'_name', Sty.commonA_open, {uuid: @props.plumbId + '_next'})
+      ep.canvas.style['z-index'] = @state.zIndex
+      eps.push(ep)
+    else
+      # make sure answer/tool endpoints are drawn *after* the div is draggable and has its endpoint
+      if (@state.task.type != 'multiple')
+        uuids = []
+        uuid = 0
+        if @state.task.type == 'single'
+          L = Object.keys(@state.task.answers).length
+        else
+          L = Object.keys(@state.task.tools).length
+        for idx in [0...L]
+          id = @getUuid(idx, uuid, uuids)
+          uuids.push(id)
+          uuid +=1
+          switch @state.task.type
+            when 'single' then ep = @props.jp.addEndpoint(id, Sty.commonA, {uuid: id})
+            when 'drawing' then ep = @props.jp.addEndpoint(id, Sty.commonA_open, {uuid: id})
+          ep.canvas.style['z-index'] = @state.zIndex
+          eps.push(ep)
+        @setUuid(null, uuid, uuids)
+      if @state.task.type != 'single'
+        ep = @props.jp.addEndpoint(@props.plumbId+'_name', Sty.commonA, {uuid: @props.plumbId + '_next'})
+        ep.canvas.style['z-index'] = @state.zIndex
+        eps.push(ep)
+    computedStyle = window.getComputedStyle(me)
+    new_state =
+      endpoints: eps
+      me: me
```

Comment:
No reason to do this. If you need the component DOM node, access it like you do above.